app/api/tool_utilities.py

```diff
@@ -14,18 +14,21 @@ def load_config():
     with open(config_path, 'r') as f:
         return json.load(f)
 
-tools_config = load_config()
+#tools_config = load_config()
 
 def get_executor_by_name(module_path):
     try:
         module = __import__(module_path, fromlist=['executor'])
-        return getattr(module, 'executor')
+        executor = getattr(module, 'executor')
```

Comment:
No need to store as a variable, we can just return

Reply:
Fixed it.